test/test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: test-secret
+stringData:
+  name: Shlok Chaudhari
+  age: "23"
+  designation: Software Engineer
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: test-deployment
+    # Mandatory field
+    app: secret-mounter
```

Comment:
@shlokchaudhari9 why do we need this label `app`?

Reply:
Right now app=secret-mounter & secret-name=<secret-name> labels both are checked to start mounting secrets on the deployment. But, @viveksinghggits pointed out that we can just drop the app=secret-mounter label and only use secret-name=<secret-name> as primary check to start mounting. I'll be doing this update as well.